### PR TITLE
Feature/change color

### PR DESCRIPTION
**Pull request by yiranli624**
## Is this a fix or a feature?
Feature

## What is the change?
When click on a category button, the color of button changes accordingly

## What does it fix? 
n/a

## Where should the reviewer start?
Line 1 of main.js; Line 58 of styles.css; Line 22 of index.html.

## How should this be tested?
By clicking each category button.

### Screenshot (optional)

@sarah01123
@elle624
@atownse

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+function returnDefaultColor(button1, category1, button2, category2) {
+    button1.classList.remove(category1);
+    button2.classList.remove(category2);
+}
+
+function changeStudyColor() {
+    changeColor(studyButton, 'study-active');
+    returnDefaultColor(meditateButton, 'meditate-active', exerciseButton, 'exercise-active');
```
Comment (atownse) [inline main.js:20]: I like that we condensed this to contain the two categories not being clicked

File: styles.css [hunk 2]
```diff
@@ -52,11 +52,39 @@ article {
...
 }
 
-.cat-img {
-  height: 2.6em;
-  padding-top: 5px;
+.study {
+  background: url(assets/study.svg) 4.35em 0.5em/2em no-repeat;
+}
+
+.study-active {
```
Comment (atownse) [inline styles.css:62]: good idea with putting these in css

**Review by atownse — COMMENTED** [review]
good work making the images change through css

**Comment by saraho1123** [comment]
Great job on the typing, Elle!

**Pull request merged**